### ISSUE TITLE
"Search Source File" results not shown

### ISSUE THREAD

**Issue by mprovvedi** [codefori/vscode-ibmi#1547]
Hi everyone,
I've a problem when searching for a string in a source file. I always get this message "No results found searching for `string searched` in `target source file`.", no matter which string I search for. But when I look in the output of the extension I can see the expected results, as you can see in this screenshot:
![image](https://github.com/halcyon-tech/vscode-ibmi/assets/107923967/ff1230cc-db07-4a6e-99d6-432815d3ade3)

Let me know if you need anything else,
Thank you for your great work!

<hr />

|Context|Version|
|-|-|
|Code for IBM i version|2.2.1|
|Visual Studio Code version|1.82.2|
|Operating System|win32_x64|

<details>
<summary>Active extensions</summary>

```
Account Microsoft (microsoft-authentication): 0.0.1
Autenticazione GitHub (github-authentication): 0.0.2
COBOL (cobol): 9.9.16
Code for IBM i Walkthroughs (vscode-ibmi-walkthroughs): 0.3.1
Collegamento automatico per debug di Node (debug-auto-launch): 1.0.0
Db2 for IBM i (vscode-db2i): 0.4.1
Emmet (emmet): 1.0.0
Error Lens (errorlens): 3.13.0
Esegui merge del conflitto (merge-conflict): 1.0.0
Funzionalità dei linguaggi TypeScript e JavaScript (typescript-language-features): 1.0.0
Funzionalità del linguaggio JSON (json-language-features): 1.0.0
GIT (git): 1.0.0
Git Base (git-base): 1.0.0
Git Graph (git-graph): 1.30.0
GitHub (github): 0.0.1
IBM i Notebooks (vscode-ibmi-notebooks): 0.0.6
Material Icon Theme (material-icon-theme): 4.30.1
Monokai Pro (theme-monokai-pro-vscode): 1.2.1
Overtype (overtype): 0.5.0
Path Intellisense (path-intellisense): 2.8.4
RPGLE (vscode-rpgle): 0.21.4
Risultato della ricerca (search-result): 1.0.0
TODO Highlight (vscode-todo-highlight): 1.0.5
WSL: Recommender (remote-wsl-recommender): 0.0.19
```
</details>

<hr />

<details>
<summary>Remote system</summary>

|Setting|Value|
|-|-|
|IBM i OS|V7R5M0|
|Tech Refresh|2|
|CCSID|1144|
|SQL|Enabled
|Source dates|Enabled

### Enabled features

|/QOpenSys/pkgs/bin|/usr/bin|/QSYS.lib/ILEDITOR.lib|/QSYS.LIB|/QIBM/ProdData/IBMiDebugService/bin|
|-|-|-|-|-|
|bash|attr|GETNEWLIBL.PGM|QZDFMDB2.PGM|startDebugService.sh|
|git|iconv||||
|ls|setccsid||||
|md5sum|tar||||
|sort|||||
|stat|||||
</details>

<details>
<summary>Shell env</summary>

```bash
```
</details>

<details>
<summary>Variants</summary>

```json
{
  "american": "#@$",
  "local": "£§$"
}
```
</details>

<details>
<summary>Errors</summary>

```json
[
  {
    "command": "system \"CHKOBJ OBJ(QSYS/QCPTOIMPF) OBJTYPE(*DTAARA)\"",
    "code": 255,
    "stderr": "CPF9801: Non trovato l'oggetto QCPTOIMPF nella libreria QSYS.",
    "cwd": "/home/USER"
  },
  {
    "command": "system \"CHKOBJ OBJ(QSYS/QCPFRMIMPF) OBJTYPE(*DTAARA)\"",
    "code": 255,
    "stderr": "CPF9801: Non trovato l'oggetto QCPFRMIMPF nella libreria QSYS.",
    "cwd": "/home/USER"
  },
  {
    "command": "export BUILDLIB=\"QGPL\" && export CURLIB=\"QGPL\" && export USERNAME=\"USER\" && export HOST=\"hostname\" && export HOME=\"/home/USER\" && export LIBLS=\"LIBRARY LIST\" && env",
    "code": 1,
    "stderr": "bsh: BUILDLIB=QGPL: is not an identifier",
    "cwd": "/home/USER"
  }
]
```
</details>

**Comment by worksofliam** [comment]
@alfax218 Now that is interesting - only happens on a 'one' source file? As in, it works on others but not a specific one??

**Comment by alfax218** [comment]
@worksofliam It works on all the others souce files I've tried so far.

**Comment by worksofliam** [comment]
@alfax218 Any chance you can share more detail about this source file it doesn't work on? Perhaps a `DSPFD OUTPUT(*PRINT)` on it?

**Comment by alfax218** [comment]
I've found something else interesting. It depends on the filter.
If there's something in the "member" section of the filter it fails, else it doesn't.

Here's an exemple.
In my previous filter i was filtering all members with "WRK*". With this condition I cannot find anything in the source file.
Then I tried to remove the filter by member name, using "*". Now I can find what I was looking for.

Also, I've tried to change the filters that preoviously were giving me results and, if I filter by member name, I get an error.
